sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/MetricsAdvisorAdministrationClient/AnomalyAlertConfigurationLiveTests.cs

```diff
@@ -318,7 +318,7 @@ public async Task CreateAndGetAlertConfigurationWithMultipleMetricConfigurations
 
             // Validate the first Metric Anomaly Alert Configuration.
```

Comment:
these are test so no big deal, but pointing them out in case you want to delete the `Anomaly` from them